README.md

```diff
@@ -20,15 +20,15 @@ The projects `CREDENTIAL_CONFIG.yml` file contains a list of variables which nee
 git clone https://github.com/<forked_repository_org>/tower_dummy_credentials
 ```
 
-2. Change the variable values in the `CREDENTIAL_CONFIG.yml` file. A list of all variables that need to be changed along with their usage can be found [HERE](VARIABLES.md).
+3. Make a copy of the `CREDENTIAL_CONFIG_TEMPLATE.yml` file and set the variables for your tower instance in the copied file `cp CREDENTIAL_CONFIG_TEMPLATE.yml CREDENTIAL_CONFIG.yml` . A list of all variables that need to be changed along with their usage can be found [HERE](VARIABLES.md).
```

Comment:
Would it be more clear if the `cp` command example was on a new line rather than in the middle of the paragraph?